Dockerfile

```diff
@@ -5,8 +5,8 @@ ARG HELM_VERSION=3.3.0
 
 WORKDIR /usr/local/bin
 
-# Install Helm, helm-gcs plugin and git
-RUN apk --update --no-cache add curl git && \
+# Install Helm, helm-gcs plugin, git and openssh-client
+RUN apk --update --no-cache add curl git openssh-client && \
```

Comment:
TODO: make a comment about why we need ssh. presumably it is because we are using SSH auth for git (which is one way to do it, but not the only way)

Reply:
As far as I'm aware, SSH auth is the only way GH supports a single repo scope. I believe HTTPS is always tied to a user's credentials (basic or token), which would effectively mean putting credentials for all repos in each individual repo build. We can explore going that path if there's a major benefit though.

Reply:
fwiw in zipkin we don't use SSH keys, but we do a little trick to stop commits from allocating to the user that generated the deploy key:

```
  # allocate commits to CI, not the owner of the deploy key
  - git config user.name "zipkinci"
  - git config user.email "zipkinci+zipkin-dev@googlegroups.com"
  # setup https authentication credentials, used by ./mvnw release:prepare
  - git config credential.helper "store --file=.git/credentials"
  - echo "https://$GH_TOKEN:@github.com" > .git/credentials
```